tox.ini

```diff
@@ -11,11 +11,12 @@ setenv =
     PYTHONHASHSEED = 42
 commands =
     python setup.py build_ext --inplace
-    py27,py34,py35: nosetests -v zarr
-    py36: nosetests -v --with-coverage --cover-erase --cover-min-percentage=100 --cover-package=zarr --with-doctest --doctest-options=+NORMALIZE_WHITESPACE zarr
+    py27: nosetests -v --with-coverage --cover-erase --cover-package=zarr zarr
+    py34,py35,py36: nosetests -v --with-coverage --cover-erase --cover-package=zarr --with-doctest --doctest-options=+NORMALIZE_WHITESPACE zarr
```

Comment:
Could you modify this so that the doctests are only run under PY36? There are some dict ordering issues which mean getting consistent doctest runs across all PY3.X versions is hard, I'm happy if doctests are only run for PY36.